ui/app/src/components/Co2Bar.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+export default class Co2Bar extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+      co2: 0.0,
+      treesCount: 0
+    };
+  }
+
+  componentDidMount() {
+    var that = this;
```

Comment:
Ist es nicht besser/sicherer var mit let zu ersetzen? Momentan ist es OK, aber wenn die Struktur in der Methode komplexer wird, könnte man that ungewollt überschreiben: https://www.youtube.com/watch?v=1sHq04erG9o